README.md

```diff
@@ -129,7 +129,7 @@ There are three types of tests for Lantern and three corresponding commands:
 |`make test_all` | runs all tests and ends if any of the tests fail| 
 |`make backup_database` | saves a database backup .sql file in the lantern base directory with name lantern_backup_`<timestamp>`.sql|
 |`make restore_database file=<backup file name>` | restores the backup database that the 'file' parameter is set to|
-
+|`make get_endpoint_resources` |Automatically queries the Epic and Cerner endpoint source websites and the NPPES npi and endpoint data and stores these resource files in the resources/prod_resources directory |
 
```

Comment:
I think it also makes sense update the steps under `Start Lantern` in this file to include, either as part of Step 2 or as a step before Step 2, to say what this command does and what files are updated. @mayerm94 might have a better idea what exactly should be specified, but i feel like it should be mentioned at least somewhere in the workflow of starting up lantern.

Reply:
I think it would make sense to mention this script in the `If you have a clean database or want to update the data in your database` section of the README before it talks about the `make populatedb` command